app/views/includes/online-tasks.nunjucks

```diff
@@ -1,14 +1,19 @@
 {% if gp.onlineTasks %}
-<ul class="link-list">
-{% if gp.onlineTasks.bookOnlineLink %}
-  <li><a class="gp-book-online" href="{{gp.onlineTasks.bookOnlineLink}}">Book a GP appointment</a></li>
-{% endif %}
-{% if gp.onlineTasks.repeatPrescriptionOnlineLink %}
-  <li><a class="repeat-prescription-online" href="{{gp.onlineTasks.repeatPrescriptionOnlineLink}}">Order a repeat prescription</a></li>
-{% endif %}
-{% if gp.onlineTasks.codedRecordsOnlineLink %}
-  <li><a class="coded-records-online" href="{{gp.onlineTasks.codedRecordsOnlineLink}}">See your health records</a></li>
-{% endif %}
+<ul class="link-list grid-row">
+  {% if gp.onlineTasks.bookOnlineLink %}
+  <li class="column--one-third">
```

Comment:
@st3v3nhunt do we have any rules for the nunjucks one? And if we don't I'll raise an issue to add it and touché with not changing it.